### DISCUSSION TITLE
Sulcal pits extraction

### DISCUSSION THREAD

**Discussion by eherrelo11** [brainvisa/brainvisa.github.io#49] [Q&A]
I would like to know if it is possible to obtain an output of the pits coordinates in 3D when using the function "sulcal pits extraction" in Brainvisa. The output Pits_texture obtained marks some points in my brain in 3D which are the coordinates I am looking for.

Thank you!

Elena.

**Comment by gauzias** [comment]
Hi, sorry for the delay.
No there is no process for that in the current version of the pipeline.
To get the coordinates, you need to write a script that will load the Pits_texture and corresponding mesh and extract the 3D coordinates of the vertices for which the value in Pits_texture is 1.
It is very easy to implement but the complexity is to manage the referential in which you want the coordinates to be extracted (e.g. the subject's native referential or the MNI/Talairach or another).
So here there are several options, and the choice depends on the analysis you have in mind.
gui
